account_cutoff_prepaid/models/account_cutoff.py

```diff
@@ -0,0 +1,180 @@
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError, ValidationError
+
+
+class AccountCutoff(models.Model):
+    _inherit = 'account.cutoff'
+
+    @api.model
+    def _get_default_source_journals(self):
+        res = []
+        cutoff_type = self._context.get('type')
```

Comment:
should also be noted to not forget change `type` here if it will be renamed in base module